dss/util/security.py

```diff
@@ -103,10 +104,17 @@ def assert_security(*decorator_args, **decorator_kwargs):
     def real_decorator(func):
         @functools.wraps(func)
         def wrapper(*args, **kwargs):
+            # Wrapper function kwargs go into decorator kwargs
             decorator_kwargs.update(kwargs)
-            decorator_kwargs['security_token'] = request.token_info
-            if decorator_kwargs.get('security_groups') is None:
```

Comment:
Moving this logic into the Authorization class, to minimize assumptions about what the wrappers will specify.

Reply:
was this comment ever addressed? i see that its moved into a sub-class of `Authorization`

Reply:
Yes! What I meant to say was, moving this logic into the `Authorization` **subclass** that requires the keywords, rather than requiring all authorization layers to define those keywords